### DISCUSSION TITLE
I want to add image size control to the rich text editor

### DISCUSSION THREAD

**Discussion by chenshangwei** [directus/directus#19056] [Feature Requests]
### Summary

Rich text editors only have a simple insert image function

### Motivation

if the image can be edited (scaled), it will be better to edit content

### Detailed Design

Click on the image , an then resize the image

### Requirements List

null

### Drawbacks

null

### Alternatives

null

### Adoption Strategy

null

**Comment by rijkvanzanten** [comment]
Please fill out the full feature request template. There isn't enough information available to reasonably discuss implementation details.
